openreview/conference/webfield.py

```diff
@@ -113,7 +113,7 @@ def set_expertise_selection_page(self, conference, invitation):
                         <b>By default, we consider all of these papers to formulate your expertise. Please click on \"Exclude\" for papers that you do  NOT want to be used to represent your expertise.</b>
                         <br>
                         <br>
-                        Your previously authored papers from selected conferences were imported automatically from <a href=https://dblp.org>DBLP.org</a>. The keywords in these papers will be used to rank submissions for you during the bidding process, and to assign submissions to you during the review process.
+                        Your previously authored papers from selected conferences were imported automatically from <a href=https://dblp.org>DBLP.org</a>. The keywords in these papers will be used to rank submissions for you during the bidding process, and to assign submissions to you during the review process. If you need to upload more <a href=https://dblp.org>DBLP.org</a> papers, you can do so by going to your OpenReview profile edit page and then clicking on 'Add DBLP Papers to Profile'.
```

Comment:
I would say:

"If you need to upload more " -> "If there are DBLP papers missing, you can go to your profiles"

@zbialecki @celestemartinez please advise.

Reply:
I would change the last part to "you can do so by editing your OpenReview profile then clicking on 'Add DBLP Papers to Profile'."

Reply:
I have included both suggestions and it now says: 

> If there are DBLP papers missing, you can add them by editing your <a href=/profile?mode=edit>OpenReview profile</a> and then clicking on 'Add DBLP Papers to Profile'.